compose/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   contracts:
     image: zkoprunet/contracts:2.0.0-beta.0
     build:
-      context: ../packages/contracts/
+      context: ../
```

Comment:
It is not works for me with this command on root
`docker-compsoe -f compose/docker-compose.yml build contracts`